### ISSUE TITLE
Songs scrobbling multiple times per album

### ISSUE THREAD

**Issue by jeandav** [newgiin/cloudplayer-scrobbler#40]
Hi! First of all thank you for this amazing extension, been using it for more than a year now and am very satisfied by it.

However it's been a few days now since the scrobbles are getting very weird, sometimes songs are not scrobbling at all and sometimes the same song (normal songs, like 3-4 minutes) are scrobbling multiple times.

I'm using it on OSX if it can help? I'm a bit familiar with javascript but I don't really know where to look into to see if I can help with anything

**Comment by ghost** [comment]
Same problem here, just look at this!

![image](https://user-images.githubusercontent.com/2155811/27537383-50480e4e-5a7c-11e7-8ad5-4ef38371e941.png)

**Comment by mylesmyles** [comment]
I had the same problem as @budkin above. I caught it early and removed all the duplicates, or else I'd submit a screenshot as well. If it happens again let us know what we can do to help/diagnose the issue! Thanks!

**Comment by ghost** [comment]
@mylesmyles right click on the extension icon, select "Options", uncheck 
```Scrobble long tracks multiple times (once every 7 minutes)?: ```, click ```Save```, reload GPM page. I don't know if this works, but so far so good - I have not had this issue for a couple of weeks

**Comment by mylesmyles** [comment]
@budkin didn't know that option existed, thanks!

**Comment by newgiin** [comment]
is this still an issue? Seems to work for me

**Comment by mylesmyles** [comment]
Yes, it's happened once more to me after using the setting @budkin recommended. Sometimes I'll use GPM on my desktop at home, leave the computer on, and use GPM from work about half an hour later. Maybe that's a unique usecase that could cause cause it to happen? To be clear, they aren't playing at the same time. I just have two instances of GPM open sometimes, with one of them playing.

If I run into the issue again, is there anything I can do to help you track down what's going on?

**Comment by newgiin** [comment]
Running two instance of GPM on one computer may cause repeated scrobbles (or actually, I would expect more missed scrobbles) but either way I'd just avoid having multiple GPM tabs open. Running on two separate computers should not matter, is that the case you're referring to?

**Comment by mylesmyles** [comment]
Yeah, that's what I'm talking about. Two different computers, but never both playing at the same time. I'm sure it's been open in multiple tabs on the same computer though, I usually have a ton of tabs open. That may have caused my issue because it was happening very randomly

**Comment by zimensky** [comment]
The issue also exists with single GPM tab (Windows 10).
Possibly cause of this is the wrong track `duration`. See issue  https://github.com/newgiin/cloudplayer-scrobbler/issues/42
